test/root.test.js

```diff
@@ -0,0 +1,21 @@
+import chai from 'chai';
```

Comment:
What is `root.test.js`?, can you find a way to structure your test to the files that they are testing